crates/simulacrum/src/lib.rs

```diff
@@ -221,19 +222,26 @@ impl<R, S: store::SimulatorStore> Simulacrum<R, S> {
 
     /// Advances the epoch.
     ///
-    /// This creates and executes an EpochChange transaction which advances the chain into the next
-    /// epoch. Since the EpochChange transaction is required to be the final transaction in an
-    /// epoch, the final checkpoint in the epoch is also created.
+    /// This creates and executes an EndOfEpoch transaction which advances the chain into the next
+    /// epoch. Since it is required to be the final transaction in an epoch, the final checkpoint in
+    /// the epoch is also created.
     ///
     /// NOTE: This function does not currently support updating the protocol version or the system
     /// packages
-    pub fn advance_epoch(&mut self) {
+    pub fn advance_epoch(&mut self, create_random_state: bool) {
```

Comment:
Why is this `create_random_state` needed? Can we add some comment explaining when to use it and when not?